.github/workflows/cluster_endtoend_upgrade.yml

```diff
@@ -94,9 +94,6 @@ jobs:
         mkdir -p /tmp/vitess-build-${{ matrix.project }}/
         cp -R bin /tmp/vitess-build-${{ matrix.project }}/
 
-    - name: Check out HEAD
-      uses: actions/checkout@v2
-
```

Comment:
Why are we removing this?

Reply:
We want to create a cluster using v13.0.0 data so we can later run the E2E test of HEAD against that cluster. This was creating a cluster using HEAD data instead of v13.0.0